raiden_contracts/contracts/TokenNetwork.sol

```diff
@@ -146,12 +146,12 @@ contract TokenNetwork is Utils {
     );
 
     modifier isOpen(uint256 channel_identifier) {
-        require(channels[channel_identifier].state == ChannelState.Opened);
+        require(channels[channel_identifier].state == ChannelState.Opened, 'Invalid channel state');
```

Comment:
I think the expected state should be mentioned in the string, e.g. 'channel state is not open'